src/store/modules/product/ProductState.ts

```diff
@@ -1,3 +1,5 @@
 export default interface ProductState {
-  cached: any;
+  cached: any,
+  products: any,
```

Comment:
Follow this:
https://github.com/hotwax/ionic-sdk/blob/main/src/store/modules/product/index.ts#L11